### ISSUE TITLE
--quick-validation flag has bad hyperparameter selection

### ISSUE THREAD

**Issue by MatthewThe** [percolator/percolator#383]
https://github.com/percolator/percolator/commit/6b325c23665d6ff64b0ff1ce28371424dcc88d17 seems to have introduced a bug in the `--quick-validation` flag. 

Previously, the best `cpos` and `cfrac` were found using a grid search in the first cross validation set. These optimal values were then reused in the remaining folds. After the commit, `cpos=1` and `cfrac=1` were hard coded for all folds. This can lead to suboptimal results.

This behavior affects v3.05 and beyond.
